Routing/Loader/DirectoryRouteLoader.php

```diff
@@ -34,6 +37,10 @@ public function __construct(RestRouteProcessor $processor)
      */
     public function load($resource, $type = null)
     {
+        if (isset($resource[0]) && '@' === $resource[0]) {
+            $resource = $this->fileLocator->locate($resource);
```

Comment:
No, this would file when a FQCN is passed to the loader because the `FileLocator` class then throws an exception.